WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeActivationBottomSheetFragment.kt

```diff
@@ -0,0 +1,41 @@
+package org.wordpress.android.ui.themes
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.View.INVISIBLE
+import android.view.View.VISIBLE
+import android.view.ViewGroup
+import androidx.annotation.NonNull
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.wordpress.android.R
+import org.wordpress.android.databinding.ThemeActivationBottomSheetBinding
+
+class ThemeActivationBottomSheetFragment : BottomSheetDialogFragment() {
+    companion object {
+        const val TAG = "ThemeActivationBottomSheetFragment"
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
```

Comment:
I noticed a minor lint issue here. You can check it by running `./gradlew ktlint`.
You can find more information on enforcing the coding style [here](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/docs/coding-style.md).

Reply:
This looks good. Thank you for the fix 🙇